libobs/util/base.c

```diff
@@ -83,6 +83,12 @@ void base_set_log_handler(log_handler_t handler, void *param)
 
 void base_set_crash_handler(void (*handler)(const char *, va_list, void *), void *param)
 {
+	if (crash_handler || crash_param) {
+		blog(LOG_ERROR, "Tried to set a crash handler when one already exists."
+				"This is unsupported, and will crash in a future version of OBS.");
```

Comment:
Do we want to promise crashing in future versions? Making this a no-op seems fine.

Reply:
Calling this a second time is now bad behavior, and we should push devs to fix it. So I‘d argue we should make it crash eventually, that would effectively make this like a deprecated method. Otherwise people might not even notice, or care enough to remove the function call (which they should).
(See e.g. bmalloc(0) as a recent-ish example, only the crashes made people really care)

Reply:
I don't disagree on the `bmalloc(0)` assessment, but I'm not sure we're ready to commit to crashing this in a future release. Maybe we'll change our opinion later.

Reply:
Removed the commit to crashing.

Reply:
Another great option would be to have a crash handling callback _after_ OBS has done it's thing.

Reply:
Would you be open to removing `exit()` call from the OBS native crash handler, so whatever code runs a crash handler afterwards would be able to pick up it's output?